### PR TITLE
KFSPTS-18904 remove PaymentWorksFormModeService

### PR DESCRIPTION
**Pull request by jhulslander**
Please note there is a related non prod SQL PR that should go with this

### CODE REVIEW

**Review by db483 — CHANGES_REQUESTED** [review]
Changes look good, I only had some small suggestions for improvements. I was also wondering if the commit message should be a bit more specific about the cleanup done rather than just "remove PaymentWorksFormModeService" (PR title)?

Thank you!

File: src/test/java/edu/cornell/kfs/pmw/batch/businessobject/PaymentWorksVendorTest.java [hunk 1]
```diff
@@ -33,35 +36,27 @@ void tearDown() throws Exception {
...
-        assertTrue("Should find the informal marketing ", StringUtils.contains(toStringValue, "informalMarketing" + 
-                connector + "false"));
-    }
-    
-    private PaymentWorksFormModeService BuildMockPaymentWorksFormModeService(boolean useForeign) {
-        PaymentWorksFormModeService service = Mockito.mock(PaymentWorksFormModeService.class);
-        Mockito.when(service.shouldUseForeignFormProcessingMode()).thenReturn(useForeign);
-        Mockito.when(service.shouldUseLegacyFormProcessingMode()).thenReturn(!useForeign);
-        return service;
+    private void assertRestringFieldExists(String toStringValue, String searchField) {
```
[the remark this answers is not in the document]
Comment (jhulslander) [reply src/test/java/edu/cornell/kfs/pmw/batch/businessobject/PaymentWorksVendorTest.java:57]: nope, typo, how about assertRestrictedFieldFormattedCorrectly

File: src/test/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksNewVendorPayeeAchServiceImplTest.java [hunk 2]
```diff
@@ -44,18 +43,9 @@ void tearDown() throws Exception {
...
-    @Test
-    void testIsUsAchBankLegacyForm() {
-        achService.setPaymentWorksFormModeService(buildMockPaymentWorksFormModeService(false));
-        boolean actualResults = achService.isUsAchBank(pmwVendor, reportData);
-        assertTrue(actualResults);
-        assertEquals(0, reportData.getRecordsThatCouldNotBeProcessedSummary().getRecordCount());
-    }
     
     @Test
     void testIsUsAchBankForeignFormUsBank() {
```
[the remark this answers is not in the document]
Comment (jhulslander) [reply src/test/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksNewVendorPayeeAchServiceImplTest.java:48]: good catch, there were a bunch of test functions that needed renamed here

File: src/main/java/edu/cornell/kfs/pmw/batch/businessobject/PaymentWorksVendor.java [hunk 3]
```diff
@@ -1267,145 +1252,34 @@ public void setPoCountry(String poCountry) {
...
+                    buildToStringAppender(bankAcctBankAccountNumber));
+            builder.append(PaymentWorksConstants.PaymentWorksVendorFieldName.BANK_ACCOUNT_BANK_ACCOUNT_NUMBER, 
+                    buildToStringAppender(bankAcctBankAccountNumber));
+            builder.append(PaymentWorksConstants.PaymentWorksVendorFieldName.BANK_ACCOUNT_BANK_VALIDATION_FILE, 
+                    buildToStringAppender(bankAcctBankValidationFile));
+            
             return builder.build();
+    }
+    
+    private String buildToStringAppender(String fieldValue) {
```
[the remark this answers is not in the document]
Comment (jhulslander) [reply src/main/java/edu/cornell/kfs/pmw/batch/businessobject/PaymentWorksVendor.java:1277]: Thanks, I switched to buildRestrictedFieldPrintableValue

**Comment by jhulslander** [comment]
this is ready for review again

**Review by db483 — APPROVED** [review]
Changes look good.

**Pull request merged**